### ISSUE TITLE
Display release conditions in more human friendly way

### ISSUE THREAD

**Issue by szpak** [szpak/CDeliveryBoy#8]
For example:
```
In release mode: true
  - in release branch: true
    - configured: 'master', actual: 'master'
    - not PR build: true
  - release triggered: true
    - not skipped by env variable ('SKIP_RELEASE'): true
    - release on demand required: true
    - release on demand triggered ('[#DO_RELEASE]'): true
  - is SNAPSHOT: true
  - current version: '0.3.2-SNAPSHOT'
  - overridden version: '0.5.0-over'
```

**Comment by szpak** [comment]
Format implemented as of 0.7.0:

```
✘ IN RELEASE MODE
  ✘ in release branch
    - configured: 'master', actual: 'master'
    ✘ not PR build
  ✔ release triggered
    ✔ not skipped by env variable ('TEST_SKIP_RELEASE')
    ✔ release on demand required
    ✔ release on demand triggered ('[#TEST_DO_RELEASE]')
  ✔ is SNAPSHOT
  - current version: '0.3.2-SNAPSHOT'
  - overridden version: '0.5.0-over'
```

**Comment by szpak** [comment]
Available in 0.7.0.
